tonic_validate/config.py

```diff
@@ -15,3 +15,4 @@
 TONIC_VALIDATE_DO_NOT_TRACK = os.getenv(
     "TONIC_VALIDATE_DO_NOT_TRACK", "false"
 ).lower() in ("true", "1", "t")
+TONIC_VALIDATE_GITHUB_ACTION = os.getenv("TONIC_VALIDATE_GITHUB_ACTION")
```

Comment:
Should we set the default value for this to False, so that it get's logged as False instead of None when someone is not using the github action?

Reply:
We don't set it since it just defaults to null on the backend. That way we can tell for sure nobody has anything set.

Reply:
Cool, sounds good.